### PR TITLE
test(taskboard, grid): add column menu tests

### PR DESCRIPTION
**Pull request by inikolova**
In this PR:
- Added test for _TaskBoard Card reordering column menu_ as per [design](https://www.figma.com/file/gWUIux95BHxemqWwOw9ZwB/Default-Theme---Internal-Usage?type=design&node-id=11393-476783&mode=design&t=gcgga45fDLc1Aze6-0);
- Added test for Grid column resizing-> [design file](https://www.figma.com/file/gWUIux95BHxemqWwOw9ZwB/Default-Theme---Internal-Usage?type=design&node-id=11671-502736&mode=design&t=Y39EI79vN4MQur7p-0);
- After sync with @dmanova, reordered items in Grid column menus. Design references:
  -- https://www.figma.com/file/gWUIux95BHxemqWwOw9ZwB/Default-Theme---Internal-Usage?type=design&node-id=11393-462009&mode=design&t=15kHIIFn9lQsv2UN-0
  -- https://www.figma.com/file/gWUIux95BHxemqWwOw9ZwB/Default-Theme---Internal-Usage?type=design&node-id=466-440&mode=design&t=Wxqn2qenGOsSFYf7-0
 - After sync with @dmanova, updated column menu items to use **_sentence case_** (following recommendations in this discussion):
   -- https://github.com/telerik/web-components-ux/issues/408#issuecomment-1327396271
   -- https://github.com/telerik/web-components-ux/issues/408#issuecomment-1195341347

### CODE REVIEW

File: tests/_output/bootstrap/grid/grid-column-resizing-actions.png
Comment (Juveniel) [inline tests/_output/bootstrap/grid/grid-column-resizing-actions.png:1]: I would suggest to remove the second Window with the checked CheckBox and disabled input, as it is rather confusing. States for the child components are already tested in the input/checkbos tests.

File: tests/_output/bootstrap/card/card-column-menu.png
Comment (Juveniel) [inline tests/_output/bootstrap/card/card-column-menu.png:1]: It is not really typicall for a Card to have a ColumnMenu. This is rather specific for the TaskBoard, thus I would suggest to add this test in the TaskBoard.

File: tests/_output/bootstrap/grid/grid-column-reordering-actions.png
Comment (Juveniel) [inline tests/_output/bootstrap/grid/grid-column-reordering-actions.png:1]: Please make sure to apply the same order of items in the other column menu tests.

**Comment by Juveniel** [comment]
@inikolova please merge the card and taskboard commits into one. I would also suggest to add the card test to [this test ](https://github.com/telerik/kendo-themes/blob/develop/tests/_output/default/taskboard/taskboard-card.png)as a new scenario.

File: packages/html/src/grid/tests/grid-column-resizing-actions.tsx [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+                        </div>
+                    </div>
+                </div>
+                <div className="k-columnmenu-item-wrapper">
+                    <div className="k-columnmenu-item">
+                        <Icon icon="group" />Autosize all columns
+                    </div>
+                </div>
+                <div className="k-columnmenu-item-wrapper">
+                    <div className="k-widget k-expander">
```
Comment (epetrow) [inline packages/html/src/grid/tests/grid-column-resizing-actions.tsx:210]: Remove `k-widget` here

**Review by JoomFX — APPROVED** [review]

**Review by epetrow — APPROVED** [review]

**Pull request merged**